functions/onprem/orborus/orborus.go

```diff
@@ -1500,7 +1500,7 @@ func main() {
 	initializeImages()
 
 	if isTenzir == "true" {
```

Comment:
Why is this here? Why would we disable a feature when it literally has a frontend component to control it entirely? This is missing the point.